fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -123,6 +123,10 @@ private String getUsername(Authentication authentication) {
     return username;
   }
 
+  public void invalidatePermission(String username) {
```

Comment:
needed during the login/logout flow to ensure we get a current user object in the fiat permission evaluator